harmony-api/src/services/topicService.ts

```diff
@@ -4,6 +4,7 @@ import * as songRepo from '../repos/songRepo'
 import * as userRepo from '../repos/userRepo'
 import { Topic, TopicId } from '../models/topic'
 import { User } from '../models/user'
+import { Album, Artist, Song } from '@prisma/client'
```

Comment:
Can we use the internal data models (or create some if they dont exist) for album/artist/song?